client/base_client.go

```diff
@@ -329,9 +329,14 @@ func (c *baseClient) getMembers(ctx context.Context, url string, timeout time.Du
 		return nil, err
 	}
 	members, err := pdpb.NewPDClient(cc).GetMembers(ctx, &pdpb.GetMembersRequest{})
-	if err != nil {
-		attachErr := errors.Errorf("error:%s target:%s status:%s", err, cc.Target(), cc.GetState().String())
-		return nil, errs.ErrClientGetMember.Wrap(attachErr).GenWithStackByCause()
+	if err != nil || members != nil && members.Header.GetError() != nil {
```

Comment:
> You can always use GetXXX() to avoid panic when using the generated proto buff struct.
When I entered the `GetHeader` function, I found that it just returned nil when resp was nil, which would cause fail if I continued to call `members.Header.GetError()` (before adding the judgment on members here, the git test had indicated an error: `point is nil`)